spec/cypress/support/admin/suppliers/commands.js

```diff
@@ -1,22 +1,12 @@
-Cypress.Commands.add("clickCreateSupplierButton", () => {
-  cy.get(".govuk-button").contains("Add a new supplier").click();
+Cypress.Commands.add("clickCreateDeliveryPartnerButton", () => {
+  cy.get(".govuk-button").contains("Add a new delivery partner").click();
```

Comment:
Hmm xD

Reply:
What?

Reply:
I think Callum likes to moan whenever we have to change cypress tests when we change copy :)

Reply:
🤷 he told me off for clicking the only button, because it would break when we added another button. Can't win 😛